README.md

```diff
@@ -143,7 +143,7 @@ This method is always run after any previous middleware.
 ```py
 def get_context(self, request):
     return {"ua": request.META.get('HTTP_USER_AGENT', ''),
-            "loggedIn": request.user.is_authenticated(),
+            "loggedIn": request.user.is_authenticated,
```

Comment:
Could you add a test to verify that the output of `get_context` is serializable. Based on experience, this will fail on Django 1.11

Reply:
This is the readme. So there's nothing to test

Reply:
Oh! My bad! I'm not used to code getting syntax highlighting readmes, I guess

Reply:
It's the fancy ` ```py ` block